static/monorail-2.0.yaml

```diff
@@ -530,6 +530,27 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+    put:
+      consumes:
+      - application/json
+      description: Modify the RackHD server configuration.
+      operationId: configPut
+      parameters:
+      - description: The configuration properties to be modified
+        in: body
+        name: config
+        required: true
+        schema:
+          $ref: '#/definitions/generic_obj'
+      responses:
+        200:
+          description: Successfully modified the configuration
+          schema:
+            type: object
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
       summary: Patch server configuration
```

Comment:
You need move summary, tags, x-authentication-type, and x-privileges back to the PATCH and then create these entries for PUT